var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -128,11 +134,11 @@ def get_cmake_args(self):
 
         if spec.satisfies('+nmodl'):
             options.append('-DCORENRN_ENABLE_NMODL=ON')
-            options.append('-DCORENRN_NMODL_ROOT=%s' % spec['nmodl'].prefix)
+            options.append('-DCORENRN_NMODL_DIR=%s' % spec['nmodl'].prefix)
             flags += ' -I%s -I%s' % (spec['nmodl'].prefix.include,
                                      spec['eigen'].prefix.include.eigen3)
 
-        nmodl_options = 'codegen --force passes --verbatim-rename --inline'
+        nmodl_options = 'codegen --force'
```

Comment:
I assume this is because we add these options by default in coreneuron.

Reply:
Exactly. In `CoreNeuron` we set as default `passes --inline`s but not `--verbatim-rename`. Is this something we should add/handle otherwise?